api/src/main/java/org/pmiops/workbench/cohortbuilder/SearchGroupItemQueryBuilder.java

```diff
@@ -72,41 +72,49 @@
       "select distinct person_id, entry_date, concept_id\n"
           + "from `${projectId}.${dataSetId}.cb_search_all_events`\n"
           + "where ";
+  private static final String STANDARD_SQL = "is_standard = %s";
```

Comment:
I was not able to comment on exact line but are  
private static final int GROUP = 1;
  private static final int NOT_GROUP = 0; used anywhere?

Reply:
Done